src/Overwatch/ResultBundle/Entity/TestResult.php

```diff
@@ -72,11 +72,11 @@ public function __toString()
         $test = $this->getTest();
 
         return
-            "[" . $this->getCreatedAt()->format("Y-m-d H:i:s") . "] " .
-            $test->getName() . " " .
-            strtoupper($this->getStatus()) . " " .
-            "(Expect " . $test->getActual() . $test->getExpectation() . $test->getExpected() . " - " .
-            $this->getInfo() . ")"
+            '[' . $this->getCreatedAt()->format('Y-m-d H:i:s') . '] ' .
+            $test->getName() . ' ' .
+            strtoupper($this->getStatus()) . ' ' .
+            '(Expect ' . $test->getActual() . $test->getExpectation() . $test->getExpected() . ' - ' .
+            $this->getInfo() . ')'
```

Comment:
Sometimes I find the `%s` soup a little less readable. This was not one of those cases :+1: